src/toolbox.ts

```diff
@@ -799,96 +824,91 @@ export class ZoomPanToolboxItem extends ToolboxItem {
+            } else if ($(event.currentTarget).hasClass("ulabel-pan-down")) {
                 annbox.scrollTop(annbox.scrollTop() + 20);
-            }
-            else if ($(event.currentTarget).hasClass("ulabel-pan-left")) {
+            } else if ($(event.currentTarget).hasClass("ulabel-pan-left")) {
                 annbox.scrollLeft(annbox.scrollLeft() - 20);
-            }
-            else if ($(event.currentTarget).hasClass("ulabel-pan-right")) {
+            } else if ($(event.currentTarget).hasClass("ulabel-pan-right")) {
                 annbox.scrollLeft(annbox.scrollLeft() + 20);
             }
-        })
+        });
 
         // Add diffrent keypress events if frames exist
         if (frames_exist) {
             $(document).on("keypress.ulabel", (event) => {
-                event.preventDefault()
+                event.preventDefault();
                 switch (event.key) {
                     case "ArrowRight":
                     case "ArrowDown":
-                        this.ulabel.update_frame(1)
-                        break
+                        this.ulabel.update_frame(1);
+                        break;
                     case "ArrowUp":
                     case "ArrowLeft":
-                        this.ulabel.update_frame(-1)
+                        this.ulabel.update_frame(-1);
                 }
-            })
-        }
-        else {
+            });
+        } else {
             $(document).on("keydown.ulabel", (event) => {
                 const annbox = $("#" + this.ulabel.config.annbox_id);
                 switch (event.key) {
                     case "ArrowLeft":
-                        annbox.scrollLeft(annbox.scrollLeft() - 20)
-                        event.preventDefault()
-                        break
+                        annbox.scrollLeft(annbox.scrollLeft() - 20);
+                        event.preventDefault();
+                        break;
                     case "ArrowRight":
-                        annbox.scrollLeft(annbox.scrollLeft() + 20)
-                        event.preventDefault()
-                        break
+                        annbox.scrollLeft(annbox.scrollLeft() + 20);
+                        event.preventDefault();
+                        break;
                     case "ArrowUp":
-                        annbox.scrollTop(annbox.scrollTop() - 20)
-                        event.preventDefault()
-                        break
+                        annbox.scrollTop(annbox.scrollTop() - 20);
+                        event.preventDefault();
+                        break;
                     case "ArrowDown":
-                        annbox.scrollTop(annbox.scrollTop() + 20)
-                        event.preventDefault()
+                        annbox.scrollTop(annbox.scrollTop() + 20);
+                        event.preventDefault();
+                        break;
                     default:
```

Comment:
ESLint wanted a `break` here from their [no-fallthrough rule](https://eslint.org/docs/latest/rules/no-fallthrough). 

Any case that has contents requires a break or a comment. Thoughts?